site/_resources/kubernetes.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Kubernetes Support Matrix
+layout: page
+---
+
+This page describes the compatibility matrix of Contour and Kubernetes versions.
+
+Contour utilizes [client-go][1] to watch for resources in a Kubernetes cluster.
+Since Kubernetes is backwards compatible with clients, older client-go versions will work with many different Kubernetes cluster versions.
+The `client-go` package includes a [compatibility matrix][2] as to what Kubernetes API versions are supported with the version of client-go.  
+
+## Supported Kubernetes versions
+
+| Kubernetes version | Contour v1.0.0 | Contour v1.0.1 | Contour v1.1.0 |
+| ------------ | :-----------: | :-----------: | :----------: | 
```

Comment:
What does the `:---:` do (sorry, being lazy and not checking)? Is the first field different on purpose?

Looks like this comes from the envoy support table, where it doesn't have any effect that I can see 🤷‍♀

Reply:
It defines the title row

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

Reply:
Oh it centers the text

https://help.github.com/en/github/writing-on-github/organizing-information-with-tables#formatting-content-within-your-table